src/main/fulcro/ui/form_state.cljc

```diff
@@ -546,6 +546,20 @@
   :args (s/cat :state map? :entity-ident util/ident? :field (s/? keyword?))
   :ret map?)
 
+(defn delete-form-state*
```

Comment:
I think remove the varargs, and make it detect a vector of idents or a single ident. That way threading will still work well with it (since you cannot easily use apply in threading).